modules/thunder_updater/updates.yml

```diff
@@ -209,3 +209,12 @@ v2.16:
         '#description_successful': '<p>The installation was successful.</p>'
         '#description_failed': '<p><strong>Installation failed</strong><p>'
         thunder_article_config: { '#text': 'Move scheduler link back from local tasks to menu tab', '#url': 'internal:/admin/config/thunder_article/configuration' }
+v2.19:
+    '#title': 'Update to Thunder 2.19'
+    '#description': '<h3>These are the new features for this release:</h3>'
+    thunder_article__thunder_update_8121:
+        '#title': 'Enable JSON-LD support'
+        '#description': '<p>JSON-LD is an implementation format for structuring data analogous to Microdata and RDFa. Typically, in terms of SEO, JSON-LD is implemented leveraging the Schema.org vocabulary, a joint effort by Google, Bing, Yahoo!, and Yandex in 2011 to create a unified structured data vocabulary for the web.</p>'
+        '#description_successful': '<p>The installation was successful.</p>'
+        '#description_failed': '<p><strong>Installation failed</strong><p>'
```

Comment:
We should give some hints, on what to do, like going to the module page and trying to enable manually.